projects/plugins/protect/src/js/components/threats-list/empty.jsx

```diff
@@ -0,0 +1,117 @@
+import { H3, Text } from '@automattic/jetpack-components';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import useProtectData from '../../hooks/use-protect-data';
+import styles from './styles.module.scss';
+
+const ProtectCheck = () => (
+	<svg width="80" height="96" viewBox="0 0 80 96" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M40 0.00634766L80 17.7891V44.2985C80 66.8965 65.1605 88.2927 44.2352 95.0425C41.4856 95.9295 38.5144 95.9295 35.7648 95.0425C14.8395 88.2927 0 66.8965 0 44.2985V17.7891L40 0.00634766Z"
+			fill="#069E08"
+		/>
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M60.9 33.6909L35.375 67.9124L19.2047 55.9263L22.7848 51.1264L34.1403 59.5436L56.0851 30.122L60.9 33.6909Z"
+			fill="white"
+		/>
+	</svg>
+);
+
+/**
+ * Time Since
+ *
+ * @param {string} date - The past date to compare to the current date.
+ * @returns {string} - A description of the amount of time between a date and now, i.e. "5 minutes ago".
+ */
+const timeSince = date => {
+	const now = new Date();
+	const offset = now.getTimezoneOffset() * 60000;
+
+	const seconds = Math.floor( ( new Date( now.getTime() + offset ).getTime() - date ) / 1000 );
+
+	let interval = seconds / 31536000; // 364 days
+	if ( interval > 1 ) {
+		return sprintf(
+			// translators: placeholder is a number amount of years i.e. "5 years ago".
+			__( '%s years ago', 'jetpack-protect' ),
```

Comment:
For singular / plural translation purposes, something like the following would be better:

```suggestion
			_n( '%s year ago', '%s years ago', Math.floor( interval ), 'jetpack-protect' ),
```

There are several other examples in the same file as well, and in `threats-list/index.jsx` from what I can see.